UnitTests/Views/TextViewTests.cs

```diff
@@ -6958,5 +6958,155 @@ public void Copy_Paste_Surrogate_Pairs ()
 			_textView.Paste ();
 			Assert.Equal ("TextView with some more test text. Unicode shouldn't 𝔹Aℝ𝔽!", _textView.Text);
 		}
+
+		[Fact, TextViewTestsAutoInitShutdown]
+		public void RuneCell_LoadRuneCells_InheritsPreviousColorScheme ()
```

Comment:
Please create a `RuneCellTests.cs` and put all runecell tests in it.

Make the tests independent of `TextView`. There should be no reason for `RuneCell` specific primitive tests to rely on `Application.Init` being called.

If there are any `TextView` specific functionalities that need testing, leave them in `TextViewTests`.

Reply:
All the existent `RuneCell` unit tests needs to use the `Application.Init` and thus, for now, there isn't needed to create a separated unit test file.

Reply:
You are not sufficiently testing RuneCell as a primitive class. The tests you have are functional/scenario tests, not unit tests. 

I'm asking you to create specific, low-level, unit tests for RuneCell. A good litmus test of good unit tests is they do not rely on anything but the class being tested itself.

Reply:
Done via https://github.com/gui-cs/Terminal.Gui/pull/2682/commits/76fad5dd4f3f5f550b029480155eb39b9b16b4ae.